project.clj

```diff
@@ -28,12 +29,15 @@
   :source-paths ["src/clj"]
   :java-source-paths ["src/java"]
 
-  :profiles {:dev {:dependencies [[puppetlabs/trapperkeeper ~tk-version
+  :profiles {:dev {:dependencies [[puppetlabs/trapperkeeper nil
                                    :classifier "test"
                                    :scope "test"]
-                                  [puppetlabs/kitchensink ~ks-version
+                                  [puppetlabs/kitchensink nil
```

Comment:
Does putting `nil` here mean it'll pull the version from the parent project? Sucks that we have to leave a `nil` lying around